MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,259 @@
+                EnableReminder="True">
+        <syncfusion:SfScheduler.BindingContext>
+            <local:SchedulerViewModel/>
+        </syncfusion:SfScheduler.BindingContext>
+        <syncfusion:SfScheduler.AppointmentMapping>
+            <syncfusion:SchedulerAppointmentMapping
+                    Subject="EventName"
+                    StartTime="From"
+                    EndTime="To"
+                    Background="Color"
+                    IsAllDay="IsAllDay"
+                    StartTimeZone="StartTimeZone"
+                    EndTimeZone="EndTimeZone"
+                    RecurrenceExceptionDates="RecurrenceExceptions"
+                    RecurrenceRule="RecurrenceRule"
+                    RecurrenceId="RecurrenceId"
+                    Reminders="Reminders">
+                <syncfusion:SchedulerAppointmentMapping.ReminderMapping>
+                    <syncfusion:SchedulerReminderMapping IsDismissed="IsDismissed"
+                    TimeBeforeStart="TimeBeforeStart"/>
+                </syncfusion:SchedulerAppointmentMapping.ReminderMapping>
+            </syncfusion:SchedulerAppointmentMapping>
+        </syncfusion:SfScheduler.AppointmentMapping>
+    </syncfusion:SfScheduler>
+{% endhighlight %}
+{% highlight c# tabtitle="ReminderViewModel.cs" %}
+public class ReminderViewModel 
+{
+  ...
+   public ObservableCollection<Event> Events { get; set; } = new ObservableCollection<Event>();
+  this.Events.Add(new Event()
+  {
+    From = DateTime.Now,
+    To = DateTime.Now.AddHours(1),
+    Color = new SolidColorBrush((Color)ColorConverter.ConvertFromString("#FF339933")),
+    EventName = "Conference",
+    Reminders = new ObservableCollection<Reminder>
+    {
+        new Reminder { TimeBeforeStart = new TimeSpan(0)},
+    }
+    });
+}
+{% endhighlight %}
+{% endtabs %}
+
+## ReminderAlertOpening event
+Scheduler notify the appointment's reminder by [ReminderAlertOpening] event before the appointment's start time. The [ReminderAlertOpeningEventArgs] has the following property,
+* [Reminders]: Gets a list of reminders that are used to notify the appointment reminders.
+
+{% tabs %}
+{% highlight c# tabtitle="MainWindow.xaml.cs" hl_lines="1" %}
+scheduler.ReminderAlertOpening += Scheduler_ReminderAlertOpening;
+
+private void Scheduler_ReminderAlertOpening(object sender, ReminderAlertOpeningEventArgs e)
+{
+    var reminders = e.Reminders;
+    e.Reminders[0].IsDismissed = true;
+}
+{% endhighlight %}
+{% endtabs %}
+## Handle Dismissed for reminder
+
+* Normal appointment directly dismissed using [IsDismissed] property
+
+* For recurrence appointment, if current occurrence need to dismiss then need to add changed occurrence for reminder occurrence dismissed, then current occurrence dismissed details get updated in underlying appointment or data source.
+
+* If only occurrence dismissed, then the changed icon will not be updated for dismissed changed occurrence
+
+## Handle Snooze for reminder
```

Comment:
Add example code for snooze and Dismissed

Reply:
Added example code for snooze and Dismissed